### PR TITLE
.inspect wrapping and column width. Closes #11 . Closes #12

### LINKED ISSUE #11: Inspect should not wrap the comments column text
```
> .inspect zendesk.zendesk_group
+------------+-----------------------------+--------------------------------+
|   Column   |            Type             |          Description           |
+------------+-----------------------------+--------------------------------+
| created_at | timestamp without time zone | The time the group was created |
| deleted    | boolean                     | True if the group has been     |
|            |                             | deleted                        |
| id         | bigint                      | Unique identifier for the      |
|            |                             | group                          |
| name       | text                        | Name of the group              |
| updated_at | timestamp without time zone | The time of the last update of |
|            |                             | the group                      |
| url        | text                        | API url of the group           |
+------------+-----------------------------+--------------------------------+
```

### LINKED ISSUE #12: Columns in text output should have a max width
The `zendesk_ticket` table goes pretty nuts now there is no max width for columns. Sideways scroll goes on a long long way.

Should we have a max column width?
Should it wrap or truncate?
What should it be? 80 chars? 120 chars? 1024 chars?

### PR DESCRIPTION
**Pull request by binaek89**

### CODE REVIEW

**Comment by binaek89** [comment]
```
> .inspect
+------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| CONNECTION | PLUGIN                                                                                                                                                                                                                                                          |
+------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| aws        | hub.steampipe.io/plugins/turbot/aws@latest                                                                                                                                                                                                                      |
| chaos      | hub.steampipe.io/plugins/turbot/chaos@latest                                                                                                                                                                                                                    |
+------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+

To get information about the tables in a connection, run '.inspect {connection}'
To get information about the columns in a table, run '.inspect {connection}.{table}'

> 
```

**Comment by binaek89** [comment]
```
> .inspect aws
+----------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| TABLE                                  | DESCRIPTION                                                                                                                                                                                                                         |
+----------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| aws_account                            | AWS Account                                                                                                                                                                                                                         |
| aws_acm_certificate                    | AWS ACM Certificate                                                                                                                                                                                                                 |
| aws_api_gateway_api_key                | AWS API Gateway API Key                                                                                                                                                                                                             |
| aws_api_gateway_authorizer             | AWS API Gateway Authorizer                                                                                                                                                                                                          |
| aws_api_gateway_rest_api               | AWS API Gateway Rest API                                                                                                                                                                                                            |
| aws_api_gateway_stage                  | AWS API Gateway Stage                                                                                                                                                                                                               |
| aws_api_gateway_usage_plan             | AWS API Gateway Usage Plan                                                                                                                                                                                                          |
| aws_api_gatewayv2_api                  | AWS API Gateway Version 2 API                                                                                                                                                                                                       |
| aws_api_gatewayv2_domain_name          | AWS API Gateway Version 2 Domain Name                                                                                                                                                                                               |
| aws_api_gatewayv2_stage                | AWS API Gateway Version 2 Stage                                                                                                                                                                                                     |
| aws_availability_zone                  | AWS Availability Zone                                                                                                                                                                                                               |
| aws_cloudformation_stack               | AWS CloudFormation Stack                                                                                                                                                                                                            |
| aws_cloudwatch_log_group               | AWS CloudWatch Log Group                                                                                                                                                                                                            |
| aws_cloudwatch_log_metric_filter       | AWS CloudWatch Log Metric Filter                                                                                                                                                                                                    |
| aws_dynamodb_backup                    | AWS DynamoDB Backup                                                                                                                                                                                                                 |
| aws_dynamodb_global_table              | AWS DynamoDB Global Table                                                                                                                                                                                                           |
| aws_dynamodb_table                     | AWS DynamoDB Table                                                                                                                                                                                                                  |
| aws_ebs_snapshot                       | AWS EBS Snapshot                                                                                                                                                                                                                    |
| aws_ebs_volume                         | AWS EBS Volume                                                                                                                                                                                                                      |
| aws_ec2_ami                            | AWS EC2 AMI                                                                                                                                                                                                                         |
| aws_ec2_application_load_balancer      | AWS EC2 Application Load Balancer                                                                                                                                                                                                   |
| aws_ec2_autoscaling_group              | AWS EC2 Autoscaling Group                                                                                                                                                                                                           |
| aws_ec2_classic_load_balancer          | AWS EC2 Classic Load Balancer                                                                                                                                                                                                       |
| aws_ec2_instance                       | AWS EC2 Instance                                                                                                                                                                                                                    |
| aws_ec2_instance_availability          | AWS EC2 Instance Availability                                                                                                                                                                                                       |
| aws_ec2_instance_type                  | AWS EC2 Instance Type                                                                                                                                                                                                               |
| aws_ec2_key_pair                       | AWS EC2 Key Pair                                                                                                                                                                                                                    |
| aws_ec2_launch_configuration           | AWS EC2 Launch Configuration                                                                                                                                                                                                        |
| aws_ec2_load_balancer_listener         | AWS EC2 Load Balancer Listener                                                                                                                                                                                                      |
| aws_ec2_network_interface              | AWS EC2 Network Interface                                                                                                                                                                                                           |
| aws_ec2_network_load_balancer          | AWS EC2 Network Load Balancer                                                                                                                                                                                                       |
| aws_ec2_target_group                   | AWS EC2 Target Group                                                                                                                                                                                                                |
| aws_ec2_transit_gateway                | AWS EC2 Transit Gateway                                                                                                                                                                                                             |
| aws_ec2_transit_gateway_route_table    | AWS EC2 Transit Gateway Route Table                                                                                                                                                                                                 |
| aws_ec2_transit_gateway_vpc_attachment |                                                                                                                                                                                                                                     |
| aws_iam_access_key                     | AWS IAM User Access Key                                                                                                                                                                                                             |
| aws_iam_group                          | AWS IAM Group                                                                                                                                                                                                                       |
| aws_iam_policy                         | AWS IAM Policy                                                                                                                                                                                                                      |
| aws_iam_role                           | AWS IAM Role                                                                                                                                                                                                                        |
| aws_iam_user                           | AWS IAM User                                                                                                                                                                                                                        |
| aws_kms_key                            | AWS KMS Key                                                                                                                                                                                                                         |
| aws_lambda_alias                       | AWS Lambda Alias                                                                                                                                                                                                                    |
| aws_lambda_function                    | AWS Lambda Function                                                                                                                                                                                                                 |
| aws_lambda_version                     | AWS Lambda Version                                                                                                                                                                                                                  |
| aws_rds_db_cluster                     | AWS RDS DB Cluster                                                                                                                                                                                                                  |
| aws_rds_db_cluster_parameter_group     | AWS RDS DB Cluster Parameter Group                                                                                                                                                                                                  |
| aws_rds_db_cluster_snapshot            | AWS RDS DB Cluster Snapshot                                                                                                                                                                                                         |
| aws_rds_db_instance                    | AWS RDS DB Instance                                                                                                                                                                                                                 |
| aws_rds_db_option_group                | AWS RDS DB Option Group                                                                                                                                                                                                             |
| aws_rds_db_parameter_group             | AWS RDS DB Parameter Group                                                                                                                                                                                                          |
| aws_rds_db_snapshot                    | AWS RDS DB Snapshot                                                                                                                                                                                                                 |
| aws_rds_db_subnet_group                | AWS RDS DB Subnet Group                                                                                                                                                                                                             |
| aws_region                             | AWS Region                                                                                                                                                                                                                          |
| aws_s3_account_settings                | AWS S3 Account Block Public Access Settings                                                                                                                                                                                         |
| aws_s3_bucket                          | AWS S3 Bucket                                                                                                                                                                                                                       |
| aws_sns_topic                          | AWS SNS Topic                                                                                                                                                                                                                       |
| aws_sns_topic_subscription             | AWS SNS Topic Subscription                                                                                                                                                                                                          |
| aws_sqs_queue                          | AWS SQS Queue                                                                                                                                                                                                                       |
| aws_vpc                                | AWS VPC                                                                                                                                                                                                                             |
| aws_vpc_customer_gateway               | AWS VPC Customer Gateway                                                                                                                                                                                                            |
| aws_vpc_dhcp_options                   |                                                                                                                                                                                                                                     |
| aws_vpc_egress_only_internet_gateway   | AWS VPC Egress Only Internet Gateway                                                                                                                                                                                                |
| aws_vpc_eip                            | AWS VPC Elastic IP                                                                                                                                                                                                                  |
| aws_vpc_endpoint                       | AWS VPC Endpoint                                                                                                                                                                                                                    |
| aws_vpc_endpoint_service               | AWS VPC Endpoint Service                                                                                                                                                                                                            |
| aws_vpc_internet_gateway               | AWS VPC Internet Gateway                                                                                                                                                                                                            |
| aws_vpc_nat_gateway                    | AWS VPC Network Address Translation Gateway                                                                                                                                                                                         |
| aws_vpc_network_acl                    | AWS VPC Network ACL                                                                                                                                                                                                                 |
| aws_vpc_route                          | AWS VPC Route                                                                                                                                                                                                                       |
| aws_vpc_route_table                    | AWS VPC Route table                                                                                                                                                                                                                 |
| aws_vpc_security_group                 | AWS VPC Security Group                                                                                                                                                                                                              |
| aws_vpc_security_group_rule            | AWS VPC Security Group Rule                                                                                                                                                                                                         |
| aws_vpc_subnet                         | AWS VPC Subnet                                                                                                                                                                                                                      |
| aws_vpc_vpn_gateway                    | AWS VPC VPN Gateway                                                                                                                                                                                                                 |
+----------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+

To get information about the columns in a table, run '.inspect {connection}.{table}'
>
```

**Comment by binaek89** [comment]
```
> .inspect aws.aws_account
+-------------------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| COLUMN                              | TYPE  | DESCRIPTION                                                                                                                                                                                                                    |
+-------------------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| account_aliases                     | jsonb | A list of aliases associated with the account, if applicable.                                                                                                                                                                  |
| account_id                          | text  | The AWS Account ID in which the resource is located                                                                                                                                                                            |
| akas                                | jsonb | Array of globally unique identifier strings (also known as) for the resource.                                                                                                                                                  |
| organization_arn                    | text  | The Amazon Resource Name (ARN) of an organization.                                                                                                                                                                             |
| organization_available_policy_types | jsonb | The Region opt-in status. The possible values are opt-in-not-required, opted-in, and not-opted-in                                                                                                                              |
| organization_feature_set            | text  | Specifies the functionality that currently is available to the organization. If set to "ALL", then all features are enabled and policies can be applied to accounts in the organization. If set to "CONSOLIDATED_BILLING", the |
|                                     |       | n only consolidated billing functionality is available.                                                                                                                                                                        |
| organization_id                     | text  | The unique identifier (ID) of an organization, if applicable.                                                                                                                                                                  |
| organization_master_account_arn     | text  | The Amazon Resource Name (ARN) of the account that is designated as the management account for the organization                                                                                                                |
| organization_master_account_email   | text  | The email address that is associated with the AWS account that is designated as the management account for the organization                                                                                                    |
| organization_master_account_id      | text  | The unique identifier (ID) of the management account of an organization                                                                                                                                                        |
| partition                           | text  | The AWS partition in which the resource is located (aws, aws-cn, or aws-us-gov)                                                                                                                                                |
| region                              | text  | The AWS Region in which the resource is located                                                                                                                                                                                |
| title                               | text  | Title of the resource.                                                                                                                                                                                                         |
+-------------------------------------+-------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
> 
```

**Comment by binaek89** [comment]
```
> select policy from aws_iam_policy limit 10
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
| POLICY                                                                                                                                                                                                                                                                        
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
| {"Statement":[{"Action":["sqs:c*","sqs:d*","sqs:g*","sqs:l*","sqs:p*","sqs:rec*","sqs:s*","sqs:t*","sqs:u*"],"Effect":"Allow","Resource":"*","Sid":"AWSsqsAdmin"}],"Version":"2012-10-17"}                                                                                    
| {"Statement":[{"Action":["acm:listc*","application-autoscaling:des*","autoscaling:des*","aws-marketplace-management:v*","aws-marketplace:g*","aws-marketplace:v*","aws-portal:v*","ce:*","cloudfront:listdistributions","cloudtrail:des*","cloudtrail:g*","cloudtrail:l*","clo
| {"Statement":[{"Action":["route53resolver:*"],"Effect":"Allow","Resource":"*","Sid":"AWSroute53ResolverAdmin"}],"Version":"2012-10-17"}                                                                                                                                       
| {"Statement":[{"Action":["acm:listc*","application-autoscaling:der*","application-autoscaling:des*","application-autoscaling:r*","autoscaling:a*","autoscaling:co*","autoscaling:createorupdatet*","autoscaling:deleten*","autoscaling:deleteta*","autoscaling:des*","autoscal
| *","events:t*","iam:g*","iam:l*","iam:si*","iam:t*","iam:un*","kms:dec*","kms:des*","kms:enc*","kms:g*","kms:l*","kms:ree*","kms:t*","kms:un*","logs:ca*","logs:createe*","logs:createlogs*","logs:deletem*","logs:des*","logs:f*","logs:getloge*","logs:listt*","logs:putl*",
| o*","sns:g*","sns:l*","sns:o*","sns:p*","sns:sets*","sns:sett*","sns:su*","sns:t*","sns:u*","sqs:ch*","sqs:deletem*","sqs:g*","sqs:l*","sqs:rec*","sqs:sen*","sqs:t*","sqs:u*","sts:*","support:*"],"Effect":"Allow","Resource":"*","Sid":"AWSOperator"}],"Version":"2012-10-1
| {"Statement":[{"Action":["acm:listc*","application-autoscaling:*","autoscaling:a*","autoscaling:co*","autoscaling:createorupdatet*","autoscaling:deleten*","autoscaling:deleteta*","autoscaling:des*","autoscaling:det*","autoscaling:di*","autoscaling:e*","autoscaling:putn*
| ep*","ec2:deletesn*","ec2:deletesp*","ec2:deletet*","ec2:deletevo*","ec2:des*","ec2:detachc*","ec2:detachn*","ec2:detachvo*","ec2:disassociatea*","ec2:disassociatei*","ec2:disassociatet*","ec2:enablet*","ec2:enablevo*","ec2:ex*","ec2:g*","ec2:importk*","ec2:modifyf*","e
| *","iam:deleter*","iam:deletese*","iam:deletess*","iam:deleteuserpe*","iam:deletev*","iam:detachg*","iam:detachr*","iam:e*","iam:g*","iam:l*","iam:pa*","iam:putg*","iam:putr*","iam:putuserpe*","iam:remover*","iam:removeu*","iam:res*","iam:s*","iam:t*","iam:un*","iam:upd
| c*","rds:modifydbc*","rds:modifydbi*","rds:modifydbp*","rds:modifydbsn*","rds:modifye*","rds:modifyo*","rds:p*","rds:reb*","rds:rem*","rds:res*","rds:s*","route53:*","route53domains:ch*","route53resolver:*","s3:a*","s3:c*","s3:deletea*","s3:deleteb*","s3:deleteo*","s3:g
| {"Statement":[{"Action":["acm:listc*","application-autoscaling:*","autoscaling:a*","autoscaling:co*","autoscaling:createorupdatet*","autoscaling:deleten*","autoscaling:deleteta*","autoscaling:des*","autoscaling:det*","autoscaling:di*","autoscaling:e*","autoscaling:putn*
| ep*","ec2:deletesn*","ec2:deletesp*","ec2:deletet*","ec2:deletevo*","ec2:des*","ec2:detachc*","ec2:detachn*","ec2:detachvo*","ec2:disassociatea*","ec2:disassociatei*","ec2:disassociatet*","ec2:enablet*","ec2:enablevo*","ec2:ex*","ec2:g*","ec2:importk*","ec2:modifyf*","e
| cing:*","ram:ac*","ram:associateresourceshare","ram:c*","ram:de*","ram:disassociateresourceshare","ram:e*","ram:getr*","ram:listpen*","ram:listpr*","ram:listresources","ram:r*","ram:t*","ram:u*","rds:ad*","rds:ap*","rds:b*","rds:co*","rds:createdbc*","rds:createdbi*","r
| s:p*","sns:s*","sns:t*","sns:u*","sqs:c*","sqs:d*","sqs:g*","sqs:l*","sqs:p*","sqs:rec*","sqs:s*","sqs:t*","sqs:u*","sts:*","support:*"],"Effect":"Allow","Resource":"*","Sid":"AWSAdmin"}],"Version":"2012-10-17"}                                                           
| {"Statement":[{"Action":["route53:*","iam:*","sts:*","route53domains:*"],"Effect":"Allow","Resource":"*"},{"Action":["route53resolver:*","ec2:*","autoscaling:*","elasticloadbalancing:*","sqs:*","events:*","cloudWatch:*","cloudTrail:*","sns:*","kms:*","logs:*","s3:*","ra
| {"Statement":[{"Action":["ec2:describereg*","ec2:describevpcs*","elasticloadbalancing:describeloadbalancers*","route53:*","route53domains:ch*","s3:listbucket"],"Effect":"Allow","Resource":"*","Sid":"AWSroute53Admin"}],"Version":"2012-10-17"}                             
| {"Statement":[{"Condition":{"StringNotLike":{"aws:RequestedRegion":["us-east*","eu*"]}},"Effect":"Deny","NotAction":["route53:*","iam:*","sts:*","route53domains:*","aws-marketplace-management:*","aws-portal:*","ce:*","pricing:*","support:*"],"Resource":"*","Sid":"BlockU
| umeTypes"}],"Version":"2012-10-17"}                                                                                                                                                                                                                                           
| {"Statement":[{"Action":["iam:List*"],"Effect":"Allow","Resource":"*"}],"Version":"2012-10-17"}                                                                                                                                                                               
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
> 
```

File: display/display.go [hunk 1]
```diff
@@ -68,13 +67,26 @@ func displayCSV(result *db.QueryResult) {
...
+
+	colConfigs := []table.ColumnConfig{}
+	headers := table.Row{}
+
+	for idx, column := range result.ColTypes {
+		headers = append(headers, column.Name())
+		colConfigs = append(colConfigs, table.ColumnConfig{
+			Name:     column.Name(),
+			Number:   idx + 1,
+			WidthMax: 1024,
```
Comment (kaidaguerre) [inline display/display.go:84]: can you move this into a const

File: metaquery/handlers.go [hunk 2]
```diff
@@ -245,14 +244,71 @@ func inspectTable(connectionName string, tableName string, input *HandlerInput)
...
+		for _, col := range row {
+			rowObj = append(rowObj, col)
+		}
+		t.AppendRow(rowObj, rowConfig)
 	}
-	table.Render()
+	t.Render()
+}
+
+func getColumnSettings(headers []string, rows [][]string) ([]table.ColumnConfig, table.Row) {
```
Comment (kaidaguerre) [inline metaquery/handlers.go:268]: what does this function do - quick comment

**Review by kaidaguerre — CHANGES_REQUESTED** [review]

File: display/display.go (hunk 1 above)
Comment (binaek89) [reply display/display.go:84]: no. it uses values from the iteration.

**Pull request merged**